docs/multicluster/architecture.md

```diff
@@ -106,3 +108,52 @@ Multi-cluster admins can also specify certain ClusterNetworkPolicies to be repli
 ClusterSet. The ACNP to be replicated should be created as a ResourceExport in the leader cluster, and
 the resource export/import pipeline will ensure member clusters receive this ACNP spec to be replicated.
 Each member cluster's Multi-cluster Controller will then create an ACNP in their respective clusters.
+
+## Antrea Multi-cluster Service Connectivity
```

Comment:
I feel "Multi-cluster Service Connectivity" is not a good term for people to understand.

How about "Multi-cluster Gateway" or "Cross-cluster Connectivity".

Reply:
Rename it to Multi-cluster Gateway"